src/components/BatteryHealthItem.tsx

```diff
@@ -0,0 +1,9 @@
+import StatsListItem from "./StatsListItem";
+
+const BatteryHealthItem = (props: { health: number }) => {
+  const health = props.health ? `${props.health}` : "--";
+
+  return <StatsListItem label="Battery Health" value={health} />;
```

Comment:
I would suggest renaming the component into `MaxCapacityItem` or so because `BatteryHealth` sounds a bit too generic.

When opening the macOS system profiler they use the label "Maximum Capacity", therefore I would also change the List Item label:

```suggestion
  return <StatsListItem label="Maximum Capacity" value={health} />;
```